lib/happo/public/happo-styles.css

```diff
@@ -8,4 +8,27 @@ body {
     #f4f4f4 10px,
     #f4f4f4 20px
   );
+  margin: 0;
+}
+
+header {
```

Comment:
Yeah, I thought about that. At this point it was just so simple that it didn't merit BEM. I saw you added it later though, that was a good call.